runtime/doc/builtin.txt

```diff
@@ -1832,6 +1834,29 @@ clearmatches([{win}])					*clearmatches()*
 		Return type: |Number|
 
 
+cmdcomplete_info([{what}])				*cmdcomplete_info()*
```

Comment:
`[{what}]` does not exist.

Reply:
Thanks. https://github.com/vim/vim/pull/17190